src/test/scala/com/fulcrumgenomics/commons/util/StringUtilTest.scala

```diff
@@ -124,4 +124,15 @@ class StringUtilTest extends UnitSpec {
       }
     }
   }
+
+  it should "split a line and place remaining text in the last value of the array when stuffLast=true" in {
```

Comment:
Maybe just loop this test for delimiter = ',' and '\t' to make sure we don't slip up on the delimiter?